### ISSUE TITLE
Why don't I get a menu link for my settings page when I generate a module?

### ISSUE THREAD

**Issue by greg-1-anderson** [Chi-teck/drupal-code-generator#55]
I was thinking about adding a template to generate the links menu, but I found that [links.menu.twig](https://github.com/Chi-teck/drupal-code-generator/blob/1.32.1/templates/d8/form/links.menu.twig) already exists. Still, the file is not generated. Is this a known issue?

Did not dig into the code.

```
$ drush generate module

 Welcome to module-standard generator!
–––––––––––––––––––––––––––––––––––––––

 Module name:
 ➤ Guest Upload

 Module machine name [guest_upload]:
 ➤ 

 Module description [The description.]:
 ➤ Allow anonymous users to upload images

 Package [Custom]:
 ➤ Other

 Dependencies (comma separated):
 ➤ block_upload

 Would you like to create install file? [Yes]:
 ➤ 

 Would you like to create libraries.yml file? [Yes]:
 ➤ no

 Would you like to create permissions.yml file? [Yes]:
 ➤ 

 Would you like to create event subscriber? [Yes]:
 ➤ 

 Would you like to create block plugin? [Yes]:
 ➤ no

 Would you like to create a controller? [Yes]:
 ➤ 

 Would you like to create settings form? [Yes]:
 ➤ 

 The following directories and files have been created or updated:
–––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
 • modules/custom/guest_upload/guest_upload.info.yml
 • modules/custom/guest_upload/guest_upload.install
 • modules/custom/guest_upload/guest_upload.module
 • modules/custom/guest_upload/guest_upload.permissions.yml
 • modules/custom/guest_upload/guest_upload.routing.yml
 • modules/custom/guest_upload/guest_upload.services.yml
 • modules/custom/guest_upload/src/Controller/GuestUploadController.php
 • modules/custom/guest_upload/src/EventSubscriber/GuestUploadSubscriber.php
 • modules/custom/guest_upload/src/Form/SettingsForm.php
```

**Comment by Chi-teck** [comment]
That template is used in `form-config` generator. The `module` generator does not generate a link for settings form because it would require asking a user three additional questions (link title, link description, link parrent). The user interaction process for module is already quite long.

**Comment by Chi-teck** [comment]
To not bother user with additinal questions we could generate some sensible values for `links.menu.twig` context.
For instance:
* link_title: Guest Upload
* link_description:  Configure guest upload
* link_parent: system.admin_config_system

**Comment by greg-1-anderson** [comment]
Yeah, that's what I was thinking -- it's easy enough for the user to find the file and customize the strings later. I agree that it would be too much prompting to ask for these values at module generation time.

**Comment by Chi-teck** [comment]
Actually that was implemented in DCG 2 long time ago. Probably needs a backport to 1.x branch.

**Comment by Chi-teck** [comment]
Fixed in DCG 1.33. Thank you.

**Comment by greg-1-anderson** [comment]
Yay! Thanks!
